test/ir_Symphony_test.cpp

```diff
@@ -81,10 +87,10 @@ TEST(TestDecodeSymphony, RealMessageDecode) {
       1198, 1284, 396, 444, 1224, 470, 1200, 470, 1198, 472};
   irsend.sendRaw(power, 23, 38000);
   irsend.makeDecodeResult();
-  EXPECT_TRUE(irrecv.decode(&irsend.capture));
+  EXPECT_TRUE(irrecv.decodeSymphony(&irsend.capture));
```

Comment:
change to decodeSymphony intentional?

Reply:
Oops. Nope. That's me debugging. Nice catch.
And this is why I like code reviews.